src/components/ItaliaTheme/manage/Widgets/CharCounterTextareaWidget.jsx

```diff
@@ -42,19 +42,17 @@ const CharCounterTextareaWidget = (props) => {
       <TextArea
         id={`field-${id}`}
         name={id}
-        value={value?.slice(0, 255) || ''}
+        value={value?.slice(0, forceMaxLength ? maxLength : 5000) || ''}
```

Comment:
```suggestion
        value={(forceMaxLength? value?.slice(0, maxLength):value) || ''}
```

Reply:
non mi è chiaro questo suggerimento di modifica... c'era qualcosa di sbagliato in quella che ho proposto, che questo sistema?

Reply:
ah capito ora ... togli qualla mia fozatura sul 5000 ... è che tutto incollato, e senza una spiegazione, non riuscivo a capire ... ok

Reply:
esatto